project-service/package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "turtle",
+  "version": "1.0.0",
+  "description": "Project service for the fastest agile project management app",
+  "main": "index.js",
+  "scripts": {
+    "start": "babel-node ./src/index.js",
+    "test": "gulp project-service:test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/the-ninja-turtles/turtle.git"
+  },
+  "author": "Andrey, David, Wesley",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/the-ninja-turtles/turtle/issues"
+  },
+  "homepage": "https://github.com/the-ninja-turtles/turtle#readme",
+  "dependencies": {
+    "body-parser": "^1.13.3",
+    "express": "^4.13.3",
+    "express-jwt": "^3.0.1",
+    "morgan": "^1.6.1",
+    "pg": "^4.4.1",
+    "pg-hstore": "^2.3.2",
+    "ramda": "^0.17.1",
+    "sequelize": "^3.6.0"
+  },
+  "devDependencies": {
+    "blue-tape": "^0.1.10",
+    "faucet": "0.0.1",
```

Comment:
are you using the faucet, tap-nyan, tap-spec, gulp-tape dependencies?

we don't need the gulp dependency here either - since it will find the node_modules in the root directory